### ISSUE TITLE
can matchBrackets highlight none-closed braces from the beginning?

### ISSUE THREAD

**Issue by marianopeck** [codemirror/CodeMirror#3247]
Hi, 
First...Thank you very much for this super cool library. Super complete! Now, let me ask. I am editing some code which may already have some dummy compilation issues...like a missing closing braces. I am using matchBrackers and it nicely highlights the none-closed braces as red (which is nice). But I want the same the first time I render the editor, without having to go with the cursor to each braces.  

In other words, when I open the editor I would like to have all none-matched braces highlighted. Is that possible? Here is an example of what I mean:

http://jsbin.com/wifozidohi/2/edit

I would like to have the 3 braces automatically highlighted, without having to give focus to it. 

Is it possible?

Thanks a lot in advance,

**Comment by marijnh** [comment]
This is a question. Ask those at http://discuss.codemirror.net, not in the bug tracker.

The answer is: no, the matchbrackets addon doesn't do this, it only shows the state of the bracket at the cursor. You'd have to write custom code (possibly using the `scanForBracket` method defined by matchbrackets) to implement the feature you describe.
